README.md

```diff
@@ -88,6 +88,16 @@ With that in mind, the steps for adding a new API are:
 4. Wait for CI to pass, fixing errors.
 5. Wait for a merge!
 
+### Test before you commit
+
+We have two automated tests running on [Travis](https://travis-ci.org/rust-lang/libc):
+
+1. `libc-test`
+  - `cd libc-test && cargo clean && cargo run`
```

Comment:
Isn't it not enough to run `rm -rf libc; rm -rf ~/.cargo; git clone https://github.com/rust-lang/libc.git` ?